jupyterlab_comments/jupyterlab_comments/handlers.py

```diff
@@ -16,8 +16,10 @@
 from jupyterlab_comments.git_commands import Git
 import traitlets.config
 import json
-git = Git(config=traitlets.config.get_config()
-         )  #global instance of connection to git commands
+from pathlib import Path
+
+home = str(Path.home())
```

Comment:
Not sure why home is declared outside the get() method of PreviousNamesHandler. Can you just call it directly?